scripts/testy_launch.py

```diff
@@ -191,6 +191,16 @@ def launch_from_distro(distro):
         instance_id = result.stdout.strip()
         wait_on_status_check(instance_id)
 
+        # Attach IAM role if requested.
+        if iam:
+            result = local(f"aws ec2 associate-iam-instance-profile \
+                --instance-id {instance_id} \
+                --iam-instance-profile Name=testy_iam", hide=True, warn=True)
```

Comment:
do you think we should allow for choosing which IAM or no?

Reply:
Yes, good idea. See 029545995482730f01632ca73d4de44f2f07c1b0